software/software.mk

```diff
@@ -25,10 +28,14 @@ HDR=$(SW_DIR)/system.h
 #SRC=$(SW_DIR)/*.c
 
 #peripherals' base addresses
-periphs.h: periphs_tmp.h
+$(SW_DIR)/periphs.h: $(SW_DIR)/periphs_tmp.h
```

Comment:
why change its place ?

Reply:
I can revert it. It is not a problem. I just thought software.mk file was a better place to put these targets.